### ISSUE TITLE
openSSL vulnerabilities with ibmjava:8-sfj-alpine and ibmjava:8-jre-alpine

### ISSUE THREAD

**Issue by kashcm** [ibmruntimes/ci.docker#49]

**Comment by kashcm** [comment]
<img width="1429" alt="screen shot 2018-12-05 at 10 55 52 am" src="https://user-images.githubusercontent.com/7726930/49529778-6e011d00-f87c-11e8-8151-cdd504298025.png">

**Comment by dnwe** [comment]
(Related: https://github.com/ibmruntimes/ci.docker/pull/28)

**Comment by bgehman** [comment]
The images are based on alpine:3.7, which have the vulnerability.  Moving to alpine:3.8 would pull in the fixes.

EDIT: correction.  alpine:3.7 has already fixed the issue (moved up to `openssl-1.0.2q-r0`) -- just need a rebuild.

**Comment by dinogun** [comment]
@kashcm Can you please confirm if you are on the latest version of the images ? I am not seeing these vulnerabilities listed [here](https://hub.docker.com/r/library/ibmjava/tags/8-jre/).

**Comment by bgehman** [comment]
@dinogun 
```
➜  ~ docker pull ibmjava:8-sfj-alpine
8-sfj-alpine: Pulling from library/ibmjava
Digest: sha256:2d77b9291ca2d7564fdd803af5e40070bcdfbb7008d473979aafc0779eb5eff9
Status: Image is up to date for ibmjava:8-sfj-alpine

➜  ~ docker run --rm -it ibmjava:8-sfj-alpine apk list | grep openssl-1.0.2
openssl-1.0.2o-r1 x86_64 {openssl} (openssl) [installed]
```

Note the insecure `openssl-1.0.2o-r1` is bundled in the ibmjava 8 **alpine** images.  The secure version is `1.0.2q-r0`, which is available in alpine's package repo: https://pkgs.alpinelinux.org/packages?name=openssl&branch=v3.7&repo=main&arch=x86_64 as of `2018-11-22 11:08:51`.  The bluemix image scanner (Vulnerability Advisor) is detecting the insecure openssl package included in these ibmjava docker hub images.

Thanks.

**Comment by bgehman** [comment]
This issue can be closed, as of the rebuild done 2 weeks ago.

```
➜  ~ docker pull ibmjava:8-sfj-alpine                                      
8-sfj-alpine: Pulling from library/ibmjava
Digest: sha256:1d09da140be806b9c27559eddf4feaa6a9fb54dfa97c4f3dc6bff19bbed502a9
Status: Image is up to date for ibmjava:8-sfj-alpine

➜  ~ docker run --rm -it ibmjava:8-sfj-alpine apk list | grep openssl-1.0.2
openssl-1.0.2q-r0 x86_64 {openssl} (openssl) [installed]
```

Latest version of openssl from Alpine is now included: `1.0.2q-r0` -- though I suspect this will continue to be an issue when new versions of openssl are released.
CC: @kashcm
